lib/api/index.js

```diff
@@ -36,6 +43,16 @@ class PublicAPI {
     return Scenarios.getCurrentMockedRequests();
   }
 
+  getAppName() {
+    return PersistentStorage.getAppName();
+  }
+
+  setAppName(appName) {
+    PersistentStorage.setAppName(appName);
+    this.init();
+    return this; // Allow chaining
```

Comment:
Can you please provide a use case where you would use chaining in bdsm?

Currently none of the API provides chaining as it does not provide any extra value in a meaningful way.